octodns/provider/ovh.py

```diff
@@ -325,7 +326,8 @@ def _is_valid_dkim(self, value):
         splitted = value.split('\\;')
         found_key = False
         for splitted_value in splitted:
-            sub_split = map(lambda x: x.strip(), splitted_value.split("=", 1))
+            sub_split = list(map(lambda x: x.strip(),
+                                 splitted_value.split("=", 1)))
```

Comment:
Let’s make this a __list comprehension__ as discussed at https://docs.python.org/3/library/functions.html#map